compiler/src/iree/compiler/Dialect/HAL/IR/HALOps.cpp

```diff
@@ -878,6 +878,10 @@ enum class NumericalType : uint32_t {
   kFloatIEEE = kFloat | 0x01,
   kFloatBrain = kFloat | 0x02,
   kFloatComplex = kFloat | 0x03,
+  kFloatish0 = kFloat | 0x04,
```

Comment:
same comment in other PR: no need to obscure this, if they can only be specific values then you can just make them specific values